frontend/uv-poc/src/app/home/home.component.ts

```diff
@@ -35,4 +41,14 @@ export class HomeComponent implements OnInit {
       }
     );
   }
+  openDialog(): void {
+    this.check = true;
+    var phoneNo = this.phoneNoForm.value.phoneNo;
+    if (phoneNo != '') {
+      const dialogRef = this.dialog.open(CustomerHistoryComponent, {
+        width: '800px',
```

Comment:
The same dialog opens with 600px in `customer-details.component.ts`

Reply:
Changed to 600px